controller/src/main/java/io/pravega/controller/server/ControllerServiceStarter.java

```diff
@@ -240,11 +240,13 @@ protected void startUp() {
                 SegmentContainerMonitorHealthContributor segmentContainerMonitorHC = new SegmentContainerMonitorHealthContributor("segmentContainerMonitor", monitor );
                 healthServiceManager.register(segmentContainerMonitorHC);
             }
-
+            String controllerServiceName = Config.getConfig("tlsCertCNName", "pravega-pravega-controller");
```

Comment:
Can we make default to "localhost"
So that if property  tlsCertCNName is not set it will fall back to original case that is localhost.

Reply:
Made default as localhost.